spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -335,4 +328,15 @@ public SubscriptionAdminClient subscriptionAdminClient(
 	public TransportChannelProvider transportChannelProvider() {
 		return InstantiatingGrpcChannelProvider.newBuilder().build();
 	}
+
+	@Configuration
+	@ConditionalOnBean(ObjectMapper.class)
```

Comment:
Hm. I thought this has to be `@CondtionalOnClass()`.
Or does this work nicely as well?

Reply:
It seems to work anyway.